server.js

```diff
@@ -1,5 +1,96 @@
-// import your node modules
+const express = require('express')
+const bodyParser = require('body-parser')
+const db = require('./data/db.js')
+const server = express()
+const PORT = 4337
+const helmet = require('helmet')
+const cors = require('cors');
 
-const db = require('./data/db.js');
+server.listen(PORT, () => console.log(`listening on port ${PORT}`));
 
-// add your server code starting here
+//add middleware
+server.use(helmet()); // call before cors and express
+server.use(cors());  // call before express
+server.use(express.json())
+server.use(bodyParser({ extended: true }))
+server.use(bodyParser.json())
+
+// server routes
+  server.get('/', (request, response) => {
+    // console.log("GET REQUEST");
+    response.send('<h1> GET REQUEST RECEIVED</h1>')
+  })  
+  
+  server.get('/api/users', (req, res) => {
```

Comment:
Our endpoint paths should be `/api/posts` since we're dealing with a database of post objects.